packages/front-end/components/Features/SDKConnections/SDKConnectionForm.tsx

```diff
@@ -104,6 +113,17 @@ export default function SDKConnectionForm({
     },
   });
 
+  const usingLatestVersion = !isSDKOutdated(
+    form.watch("languages")?.[0] || "other",
+    form.watch("sdkVersion")
+  );
+
+  const useLatestSdkVersion = () => {
+    const language = form.watch("languages")?.[0] || "other";
+    const latest = getCurrentSDKVersion(language);
```

Comment:
Is `getCurrentSDKVersion` grabbing the SDK version for the user or the latest available version for the language? If it's the latter I think the function should be called `getLatestSDKVersion`